src/core.rs

```diff
@@ -713,6 +713,265 @@ impl Tox {
+        conference_number: u32,
+        peer_number: u32
+    ) -> Result<String, ConferencePeerQueryError> {
+        unsafe {
+            let size = tox_try!(err, ll::tox_conference_peer_get_name_size(
+                self.raw,
+                conference_number,
+                peer_number,
+                &mut err
+            ));
+
+            let mut name = Vec::with_capacity(size);
+
+            tox_try!(err, ll::tox_conference_peer_get_name(
+                self.raw,
+                conference_number,
+                peer_number,
+                name.as_mut_ptr(),
+                &mut err
+            ));
+
+            Ok(String::from_utf8_unchecked(name))
+        }
+    }
+
+    pub fn get_peer_public_key(
+        &mut self,
+        conference_number: u32,
+        peer_number: u32,
+    ) -> Result<PublicKey, ConferencePeerQueryError> {
+        unsafe {
+            let mut raw: [u8; PUBLIC_KEY_SIZE] = mem::uninitialized();
+
+            tox_try!(err, ll::tox_conference_peer_get_public_key(
+                self.raw,
+                conference_number,
+                peer_number,
+                raw.as_mut_ptr(),
+                &mut err
+            ));
+
+            Ok(PublicKey {
+                raw
+            })
+        }
+    }
+
+    pub fn is_own_peer_number(
+        &mut self,
+        conference_number: u32,
+        peer_number: u32
+    ) -> Result<bool, ConferencePeerQueryError> {
+        unsafe {
+            let is_ours =
+                tox_try!(err, ll::tox_conference_peer_number_is_ours(
+                    self.raw,
+                    conference_number,
+                    peer_number,
+                    &mut err
+                ));
+
+            Ok(is_ours)
+        }
+    }
+
+    pub fn conference_offline_peer_count(
+        &mut self,
+        conference_number: u32
+    ) -> Result<u32, ConferencePeerQueryError> {
+        unsafe {
+            let count = tox_try!(err, ll::tox_conference_offline_peer_count(
+                self.raw,
+                conference_number,
+                &mut err
+            ));
+
+            Ok(count)
+        }
+    }
+
+    pub fn get_offline_peer_name(
+        &mut self,
+        conference_number: u32,
+        peer_number: u32,
+    ) -> Result<String, ConferencePeerQueryError> {
+        unsafe {
+            let size = tox_try!(err, ll::tox_conference_offline_peer_get_name_size(
+                self.raw,
+                conference_number,
+                peer_number,
+                &mut err
+            ));
+
+            let mut name = Vec::with_capacity(size);
+
+            tox_try!(err, ll::tox_conference_offline_peer_get_name(
+                self.raw,
+                conference_number,
+                peer_number,
+                name.as_mut_ptr(),
+                &mut err
+            ));
+
+            Ok(String::from_utf8_unchecked(name))
+        }
+    }
+
+    pub fn get_offline_peer_public_key(
```

Comment:
This is a misleading name. It looks like you want to get an offline peer's PK (friend's), however you are dealing with conferences. TokTok's naming is fine, it starts with `conference_` which gives us a point of what you are working with, and the latter `peer_get_public_key` - what you want from it. I think you should take their naming convention.

Reply:
> I think you should take their naming convention.

This would require to change all the names in the library to keep naming consistent. Beside that, I wanted names to be more human-like.

Reply:
Up to you